Documentation/Blazorise.Docs/Blazorise.Docs.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
+﻿<Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
```

Comment:
As I mentioned. This project is now the client-side project.

Reply:
From watching the day 1 from net conf, the reason for this is that when we are requesting for WASM components, this tells the application that this is a valid project to be downloaded as a DLL. 

This is because, as opposed to before you have an explicit server app, that is able to serve/orchestrate wasm simultaneously, so it needs to figure out what DLLs should be sent to the client. So apparently the `<Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">` marker is what indicates this.

So this does make sense if you know your razor library is going to be consumed as WASM.